src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -1011,6 +1018,53 @@ void WindowImplWin32::processEvent(UINT message, WPARAM wParam, LPARAM lParam)
 
             break;
         }
+
+        // File(s) dropped event
+        case WM_DROPFILES:
+        {
+            const HDROP hDrop = (HDROP)wParam;
```

Comment:
```suggestion
            const HDROP hDrop = static_cast<HDROP>(wParam);
```
I'm not entirely sure `static_cast` is appropriate but in any case, please stop using this C-style cast.